pkg/sdk/streamlits_def.go

```diff
@@ -103,7 +103,10 @@ var StreamlitsDef = g.NewInterface(
 		OptionalLike().
 		OptionalIn().
 		OptionalLimit(),
-).ShowByIdOperation().DescribeOperation(
+).ShowByIdOperation(
```

Comment:
both were added to def using SchemaOBjectIdentifier, let's check how it behaves for:
- accountObjectIdentifier
- databaseObjectIdentifier

Reply:
generated the examples:
* accountObjectIdentifier - connections with `Like` filtering
* databaseObjectIdentifier - application_roles with `ApplicationName` filtering as this object does not implement common `Like` or `In` filtering